.claude/agents/pr-review-handler.md

```
@@ -0,0 +1,599 @@
+
+# Extract actionable comments section
+gh pr view <PR_NUMBER> --repo cloudposse/atmos --comments | \
+  grep -A 100 "Actionable comments posted"
+
+# Only use code diffs as reference, not for direct application
+gh pr view <PR_NUMBER> --repo cloudposse/atmos --comments | \
+  grep -A 20 "Committable suggestion"
+```
+
+#### Example AI Prompt from CodeRabbit:
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)